js/components/thumbnailCarousel.js

```diff
@@ -169,34 +162,29 @@ var ThumbnailCarousel = React.createClass({
       centerWidth: this.state.centerThumbnailWidth,
       scrubberBarWidth: this.props.scrubberBarWidth,
       top: this.state.centerThumbnailHeight - this.state.thumbnailHeight,
-      pos: centralThumbnail.pos,
+      pos: this.props.centralThumbnail.pos,
       padding: this.state.thumbnailPadding
     };
 
     var thumbnailsBefore = this.findThumbnailsBefore(data);
     var thumbnailsAfter = this.findThumbnailsAfter(data);
     var thumbnailClassName = "oo-thumbnail-carousel-center-image";
-    var thumbnailStyle = {
-      left: (data.scrubberBarWidth - data.centerWidth) / 2
-    };
-    if (Utils.isValidString(centralThumbnail.url)) {
-      thumbnailStyle.backgroundImage = "url('" + centralThumbnail.url + "')";
-    }
 
     if (this.props.videoVr) {
-      thumbnailStyle.backgroundSize = this.props.imageWidth + "px " + this.props.imageHeight + "px";
-      thumbnailStyle.backgroundPosition = this.props.positionX + "px " + this.props.positionY + "px";
-      thumbnailStyle.repeat = "repeat no-repeat";
       thumbnailClassName += " oo-thumbnail-vr";
     }
 
-    var time = isFinite(parseInt(this.props.hoverTime)) ? Utils.formatSeconds(parseInt(this.props.hoverTime)) : null;
+    var thumbnailStyle = {};
+    if (this.props.thumbnailStyle !== null && typeof this.props.thumbnailStyle === 'object') {
+      thumbnailStyle = this.props.thumbnailStyle;
+    }
+    thumbnailStyle.left = (data.scrubberBarWidth - data.centerWidth) / 2;
```

Comment:
This will affect this.props.thumbnailStyle. Is this ok?

Reply:
Yes this is ok. We wanted to change the 'left' property of the this.props.thumbnailStyle object